hugo/layouts/partials/home/our-links.html

```diff
@@ -1,10 +1,6 @@
-<p>
-    {{ $home := $.Site.Data.home.home }}
-    {{ range where $home.hero.links "url" "ne" "#"  }}
```

Comment:
It was an existing and useful behavior in the past, and I'm not sure if it won't be used any more (I believe that it's more likely that they want to remove one button in the future, than they want to keep both forever and ever)
Anyway, if you find it a waste, I'm ok if you drop that feature, of course (I just wanted to point it out).